domain/AdminProperties.go

```diff
@@ -315,7 +315,7 @@ func (d *Domain) AdminProperties(in *AdminPropertiesIn) (out AdminPropertiesOut)
 		fallthrough
 	case zCrud.CmdList:
 		r := rqProperty.NewProperty(d.PropOltp)
-		out.Properties = r.FindByPagination(&AdminPropertiesMeta, &in.Pager, &out.Pager)
+		out.Properties = r.FindByPaginationWithNote(&AdminPropertiesMeta, &in.Pager, &out.Pager)
```

Comment:
di AdminPropertiesMeta perlu dikasih 
			{
				Name:      `about`,
				Label:     `About`,
				DataType:  zCrud.DataTypeString,
				InputType: zCrud.InputTypeText,
			},
^ contact* juga

dan waktu display jangan pakai arrayOfArray = true, tapi false

Reply:
yg bagian atas waktu upsert/delete/restore
terpaksa struct In-nya nerima PropertyWithNote bukan lagi Property

lalu di dalamnya note-nya yg di json.Marshal dari about dan contact*

Reply:
atau ini ngga usah, yg penting waktu display sih ga dipecah note-nya, biar ga perlu kode frontend/backend buat gabung sama sekali, jadi admin inputin beneran json

Reply:
ooh iya, sip
berarti nambahin meta

Reply:
tapi yang pasti gk ada search by note/contact/email ya di admin/properties

Reply:
iyes, search ya search `raw about":"bla`